cross/rest-server/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME = rest-server
```

Comment:
Please name the package restic-rest-server (and rename the folder)

To have a variable for the downloaded file as rest-server, I propose to use an additional variable like PKG_REAL_NAME = rest-server. And use this new variable instead.

the name for the spk package should be restic-rest-server or restic (rest-server is just too common).

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/5550/commits/d176e86f04b4daaefd98357099096a53de51a68b